src/networking.c

```diff
@@ -2765,11 +2765,15 @@ static void setProtocolError(const char *errstr, client *c) {
         char buf[256];
         buf[0] = '\0';
         if (c->querybuf && sdslen(c->querybuf) - c->qb_pos < PROTO_DUMP_LEN) {
-            snprintf(buf, sizeof(buf), "Query buffer during protocol error: '%s'", c->querybuf + c->qb_pos);
+            snprintf(buf, sizeof(buf), "Query buffer during protocol error: '%s'",
+                     server.hide_user_data_from_log ? "[hide_user_data_from_log]Redacted" : (c->querybuf + c->qb_pos));
```

Comment:
I think we can just print `*redacted*` like we do in other places.

```suggestion
            snprintf(buf, sizeof(buf), "Query buffer during protocol error: '%s'",
                     server.hide_user_data_from_log ? "*redacted*" : (c->querybuf + c->qb_pos));
```